metaflow/metaflow_config.py

```diff
@@ -262,6 +262,20 @@
 ARGO_WORKFLOWS_KUBERNETES_SECRETS = from_conf("ARGO_WORKFLOWS_KUBERNETES_SECRETS", "")
 ARGO_WORKFLOWS_ENV_VARS_TO_SKIP = from_conf("ARGO_WORKFLOWS_ENV_VARS_TO_SKIP", "")
 
+##
+# Argo Events Configuration
+##
+ARGO_EVENTS_SERVICE_ACCOUNT = from_conf(
+    "ARGO_EVENTS_SERVICE_ACCOUNT", "operate-workflow-sa"
+)
+ARGO_EVENTS_EVENT_BUS = from_conf("ARGO_EVENTS_EVENT_BUS", "default")
+ARGO_EVENTS_EVENT_SOURCE = from_conf("ARGO_EVENTS_EVENT_SOURCE", "metaflow-webhook")
+ARGO_EVENTS_EVENT = from_conf("ARGO_EVENTS_EVENT", "event")
+ARGO_EVENTS_WEBHOOK_URL = from_conf(
+    "ARGO_EVENTS_WEBHOOK_URL", "http://10.10.29.11:12000/event"
```

Comment:
nit: is this a standard URL or left over from testing?

Reply:
The PR is *not* yet ready to be merged. There are pending changes for `metaflow configure` etc.